pkg/controller/keystoneservice/keystoneservice_controller.go

```diff
@@ -86,9 +87,28 @@ func (r *ReconcileKeystoneService) Reconcile(request reconcile.Request) (reconci
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling KeystoneService")
 
+	keystoneAPI := keystone.API(request.Namespace, "keystone")
```

Comment:
This hard codes the name of the API to keystone though? I think we'll have to search for keystone objects in this namespace right and use the one we find...

Reply:
Going to go ahead and merge this. I will add a comment which identifies that identifies this as being hard coded for now.